pytorch_lightning/plugins/precision/mixed.py

```diff
@@ -11,16 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import TYPE_CHECKING, Union
+from typing import Union
 
 from pytorch_lightning.plugins.precision.precision_plugin import PrecisionPlugin
-
-if TYPE_CHECKING:
-    from pytorch_lightning.utilities import AMPType
+from pytorch_lightning.utilities import AMPType
 
 
 class MixedPrecisionPlugin(PrecisionPlugin):
     """Base Class for mixed precision."""
 
-    backend: "AMPType"
+    @property
+    def backend(self) -> AMPType:
+        """.. deprecated:: v1.6 This property is deprecated in 1.6 and will be removed in 1.7. Please use instance
```

Comment:
some issue with docstrings here :)

Reply:
yes, just fixed that. pre-commit kept reformatting these to the format sphinx could not read -.-